client/src/components/GlobalStyledComponents/ReversimStyledComps.jsx

```diff
@@ -30,7 +30,16 @@ export const AlignCenter = styled.div`
         max-width: 1280px;
       };    
     `}
-`
+`;
+export const FlexRow = styled.div`
```

Comment:
Naming wise, I wrote about it in another pull request, I think `Flex` is an implementation detail. We should call it simply `Row`. The name should describe what it does not what it uses to do that.